src/java_1/dataTypes/MyArrays.java

```diff
@@ -17,18 +20,33 @@ void greet(String name) {
                "ABRAM" => greet() => "Hola Abram"
                "maTT => greet() => "أهلا و سهلا Matt"
  */
+        Random random = new Random();
+        int index = random.nextInt(greetings.length);
+
+        System.out.println(greetings[index] +" " + name);
+
     };
 
-    boolean[] bitsToBinaries() {
+    ArrayList<Boolean> bitsToBinaries(int[] bits) {
```

Comment:
This method should convert an array of 1's and 0's to a binary (boolean) array with the appropriate "true" or "false" printing for each matching value.